### PR TITLE
Add ssh key facts for hcloud

### PR DESCRIPTION
**Pull request by ghost**
##### SUMMARY
Adds a hcloud_ssh_key_facts module

##### ISSUE TYPE
- New Module Pull Request

##### COMPONENT NAME
hcloud_ssh_key_facts

##### ADDITIONAL INFORMATION
Follow up of #53550 and #52553.

### CODE REVIEW

**Review by LKaemmerling — APPROVED** [review]

**Comment by LKaemmerling** [comment]
shipit

File: lib/ansible/modules/cloud/hcloud/hcloud_ssh_key_facts.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+        name:
+            description: Name of the ssh_key
+            returned: always
+            type: str
+            sample: my-ssh-key
+        fingerprint:
+            description: Fingerprint of the ssh key
+            returned: always
+            type: str
+            sample: running
```
Comment (resmo) [inline lib/ansible/modules/cloud/hcloud/hcloud_ssh_key_facts.py]: fingerprint is running? rofl

Comment (LKaemmerling) [inline lib/ansible/modules/cloud/hcloud/hcloud_ssh_key_facts.py]:
```suggestion
            sample: b7:2f:30:a0:2f:6c:58:6c:21:04:58:61:ba:06:3b:2f
```

**Review by LKaemmerling — CHANGES_REQUESTED** [review]

**Review by resmo — APPROVED** [review]
shipit

**Comment by resmo** [comment]
bot_status

**Comment by gundalow** [comment]
Should this be `_info` module https://github.com/ansible/ansible/issues/54280

**Comment by cschmitt-hcloud** [comment]
@gundalow Just out of curriosity, this module is returning `ansible_facts` and based on the rules in youre mentioned issue this would make it a fact module, or did I miss something?

**Review by LKaemmerling — APPROVED** [review]

**Comment by gundalow** [comment]
Thanks all, merged

**Pull request merged**